101-workshop-nodejs/ex0-what-is-serverless/serverless-characteristics.md

```diff
@@ -17,55 +17,45 @@
 #
 -->
 
-# Characteristics of Serverless
+# Characteristics of serverless
 
-## Taking the "Ops" out of "DevOps"
+## Taking the Ops out of DevOps
 
-In today's agile-focused world, Developers are now (more than ever) expected to understand the operational considerations of deploying, hosting and maintaining their applications "on the Cloud"; hence the term "DevOps".
+In today's agile-focused world, developers are now, more than ever, expected to understand the operational considerations of deploying, hosting, and maintaining their applications on the cloud. This often means developers spend less time coding applications and much more time worrying about the operational aspects.
 
-However, this often means less time actually coding the application and much more time worrying about the operational aspects.
+![The DevOps blues](images/101-ex0-serverless-devops-blues.png)
 
-![The "DevOps" blues](images/101-ex0-serverless-devops-blues.png)
+## NoOps for developers
 
-## “→ NoOps” for the Developer
```

Comment:
This is a programming (math) symbol for "approaching" it was meant to be part of "is a" and another programmer reference.

Reply:
Oh! Okay. I don't think there's a need for quotation marks.